frontend/src/nc/pages/components/AddModal.js

```diff
@@ -43,6 +43,11 @@ const AddModal = ({ setModal, table, columns_, postEndpoint, refreshFunc }) => {
       const email = JSON.parse(localStorage.getItem("user"))?.email;
```

Comment:
should the extension of this file be jsx?

Reply:
Done